### PR TITLE
M2861 Table attribute expand/collapse + remove form plugin

### PR DESCRIPTION
**Pull request by dennishendriksen**
- React Table component (replaces jquery.molgenis.table)
  - Column expand and collapse
- Dialog, Popover, Pager components
- Remove Form plugin (deprecated by data explorer in combination with React Table)

### CODE REVIEW

**Comment by Mark-de-Haan** [comment]
Setting a filter on an xcategorical_value.value (TypeTest) throws an error

screen shot 2015-06-05 at 16 12 57

**Comment by Mark-de-Haan** [comment]
The create button at the end of the Create <entity_name> form does not do anything.
Pressing it multiple times did not work, there is no loading or request being done

screen shot 2015-06-05 at 16 46 22

**Comment by Mark-de-Haan** [comment]
When I deselect all attributes, I still see the edit / delete / zoom in buttons which I think looks a bit weird

screen shot 2015-06-05 at 16 54 49

**Comment by Mark-de-Haan** [comment]
Downloading in any form throws an error

screen shot 2015-06-05 at 17 00 56

**Comment by Mark-de-Haan** [comment]
Setting a filter on xtext throws an error

screen shot 2015-06-05 at 17 07 33

**Comment by Mark-de-Haan** [comment]
This fixes #2754

**Comment by dennishendriksen** [comment]
This does not fix #2754, try filtering on the id column of TypeTest.

**Comment by Mark-de-Haan** [comment]
Cannot compile, Run clean install with tests to find the source

File: molgenis-data-import/src/main/java/org/molgenis/data/exporter/EmxExporterService.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	// try
+	// {
+	// entitiesWritable.close();
+	// }
+	// catch (IOException e)
+	// {
+	// throw new RuntimeException(e);
+	// }
+	// }
+	// }
```
Comment (Mark-de-Haan) [inline molgenis-data-import/src/main/java/org/molgenis/data/exporter/EmxExporterService.java]: Remove Commented code

**Pull request merged**